src/engraving/dom/undo.cpp

```diff
@@ -795,6 +795,7 @@ void CloneVoice::redo(EditData*)
 
 AddElement::AddElement(EngravingItem* e)
 {
+    DO_ASSERT_X(!e->generated(), String("Generated item %1 passed to AddElement").arg(String::fromAscii(e->typeName())));
```

Comment:
In this case I think 
```suggestion
    DO_ASSERT_X(!e->generated(), String(u"Generated item %1 passed to AddElement").arg(String::fromAscii(e->typeName())));
```